### DISCUSSION TITLE
Feature Inquiry

### DISCUSSION THREAD

**Discussion by omicreativedev** [danielBingham/peerreview#129] [Ideas]
Good day!

I'm not a programmer or a Phd. I'm just a student who does a lot of research. :)

Wondering if or how you'll be handling / adding citation generation? It would be awesome if made compatible with the top citation software. I personally use Zotero but also Noodle Tools, Bibex, whatever else- in a variety of formats. Some are available right from Zotero itself but at least APA, MLA, the main ones for students.

Also wondering if you'll have internal annotation tools for readers. I'm currently using Zotero or Hypothes.is but it's nice when some systems have a way for students to annotate and then download their highlights with their citations.

Will there be accounts? If so, what about favorites and collections? 

It might also be worthwhile to have a method of updating missing citations- both ways. I know a lot of authors struggle when someone has cited their paper. 

Just some features that I think would make the platform attractive from the learner point of view.

**Comment by danielBingham** [comment]
Hey @omicreativedev !  Thanks for all the ideas and suggestions.  

There are accounts and anyone can create them (you're welcome to explore the alpha if you'd like: https://blog.peer-review.io/ready-for-alpha-testing). Citation generation, platform/tooling integrations, and collections/favorites are all things that are potentially on the horizon.  Though I'm not sure how soon.  

At the moment, we're laser focused on the features necessary to get people out of the journal system and publishing through the platform.  If we manage to achieve that (get on a good adoption curve and have enough funding to continue to maintain it and build on it) then we'll be able to step back and consider what other features would be really helpful for everyone.

In addition to those features, we've also thought about things like linking replications to the papers being replicated, providing a space to share data and attach it to the final paper, study registration, a comment/discussion area for non-scholars where they can ask questions about papers, community moderation tooling (and moderation abilities that are gained with reputation), and how to use the platform's reputation system to foster a supportive, collaborative environment.  There are a lot of possible directions and we're eager to explore them!
